.github/workflows/build_and_test.yml

```diff
@@ -71,17 +73,37 @@ jobs:
              export IPDK_RECIPE=`pwd`
              cd $IPDK_RECIPE/setup
              cmake -B build -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/deps
-             cmake --build build -j6
+             cmake --build build -j6   
 
+      - name: 'Run Stratum unit tests'
+        run: |
+             cd /tmp
+             curl -o bazel-4.2.3-installer-linux-x86_64.sh https://github.com/bazelbuild/bazel/releases/download/4.2.3/bazel-4.2.3-installer-linux-x86_64.sh
```

Comment:
Can we add a log here saying "Installing Bazel"?

Reply:
done!